variables.tf

```diff
@@ -94,6 +94,12 @@ variable "existing_kms_instance_guid" {
   default     = null
 }
 
+variable "existing_kms_instance_crn" {
```

Comment:
We already ask for `existing_kms_instance_guid` - so if you now need to CRN, you should remove the `existing_kms_instance_guid` variable and parse the GUID from the CRN. No point in asking consumer to pass both